### DISCUSSION TITLE
Docker-OSX Helm Chart & Kubernetes Supported

### DISCUSSION THREAD

**Discussion by sickcodes** [sickcodes/Docker-OSX#167] [Ideas, Suggestions, Requests]
I would like to give a personal thanks to @cephasara for creating the Helm charts for Docker-OSX to enable full blown Kubernetes support.

https://github.com/sickcodes/Docker-OSX/pull/124

https://github.com/sickcodes/Docker-OSX/pull/125

https://github.com/sickcodes/Docker-OSX/pull/132

https://github.com/sickcodes/Docker-OSX/pull/134

You can use it now here: https://artifacthub.io/packages/helm/docker-osx/docker-osx

**Comment by marcellodesales** [comment]
@cephasara @sickcodes I'm looking for a quick start guide on setting up a CI/CD instance on Gitlab CI or Github Actions... All information is all over the place... Is there any work I can contribute?
If I run Gitlab in Kubernetes, I should be able to run the runners using this approach... I'm looking for instructions on how to set it up in such a way that all runners will use the same docker image with the XCode setup to build IOS apps...

**Reply by sickcodes** [reply]
When you're happy with your image you can Docker commit that image.

**Reply by nomansadiq11** [reply]
@marcellodesales  and @sickcodes  any update on this ?
